RevenueCatUI/UIKit/PaywallViewController.swift

```diff
@@ -112,6 +118,10 @@ public protocol PaywallViewControllerDelegate: AnyObject {
     optional func paywallViewController(_ controller: PaywallViewController,
                                         didFinishRestoringWith customerInfo: CustomerInfo)
 
+    /// Notifies that the ``PaywallViewController`` was dismissed.
+    @objc(paywallViewControllerDismissed:)
+    optional func paywallViewControllerDismissed(_ controller: PaywallViewController)
```

Comment:
This makes sense 👍🏻 
I'd rename this though:
```suggestion
    optional func paywallViewControllerWasDismissed(_ controller: PaywallViewController)
```